### PR TITLE
Claudio/performance counter bump drun

### PR DESCRIPTION
**Pull request by crusso**

### CODE REVIEW

File: nix/drun.nix [hunk 1]
```diff
@@ -15,7 +15,8 @@ pkgs:
       # installed. You will normally not be bothered to perform
       # the command therein manually.
 
-      cargoSha256 = "sha256-Oi7hKhif0CpnovGNX+/hpKzfMDuSpoWwtmH0WbnrLqg=";
+      cargoSha256 = "sha256-EbYM5DxJILstvNjQS4Jr9RvCxsT71XfcZ4lWzBiJaTU=";
+      # "sha256-Oi7hKhif0CpnovGNX+/hpKzfMDuSpoWwtmH0WbnrLqg=";
```
Comment (crusso) [inline nix/drun.nix:19]:
```suggestion
```

File: src/codegen/compile.ml [hunk 2]
```diff
@@ -3671,6 +3672,14 @@ module IC = struct
         edesc = nr (FuncExport (nr (E.built_in env "print_ptr")))
       })
 
+
+  let performance_counter env =
+    match E.mode env with
+    | Flags.ICMode | Flags.RefMode ->
```
Comment (ggreif) [inline src/codegen/compile.ml:3678]:
```suggestion
    | Flags.(ICMode | RefMode) ->
```

**Review by ggreif — APPROVED** [review]
Cool!

File: src/codegen/compile.ml (hunk 2 above)
Comment (crusso) [reply src/codegen/compile.ml]: We use this style elsewhere...

Comment (ggreif) [reply src/codegen/compile.ml]: My conception is that it makes the two relevant pieces  of information appear nearer and is thus better digestible. But I am not radical enough about it to change all occurrences.

**Pull request merged**